packages/cli/src/commands/lint.ts

```diff
@@ -197,20 +211,30 @@ const lintCommand: CommandModule = {
         results = filterResultsBySeverity(results, failSeverity);
       }
 
+      const scoringConfigData = getScoringConfig(scoringConfig);
+
       await Promise.all(
         format.map(f => {
-          const formattedOutput = formatOutput(results, f, { failSeverity: getDiagnosticSeverity(failSeverity) });
+          const formattedOutput = formatOutput(results, f, {
+            failSeverity: getDiagnosticSeverity(failSeverity),
+            scoringConfig: scoringConfigData,
+          });
           return writeOutput(formattedOutput, output?.[f] ?? '<stdout>');
         }),
       );
 
       if (results.length > 0) {
-        process.exit(severeEnoughToFail(results, failSeverity) ? 1 : 0);
+        process.exit(
+          scoringThresholdNotEnough(results, scoringConfigData) ? 1 : severeEnoughToFail(results, failSeverity) ? 1 : 0,
+        );
       } else if (config.quiet !== true) {
         const isErrorSeverity = getDiagnosticSeverity(failSeverity) === DiagnosticSeverity.Error;
         process.stdout.write(
           `No results with a severity of '${failSeverity}' ${isErrorSeverity ? '' : 'or higher '}found!\n`,
         );
+        if (scoringConfig !== void 0) {
+          process.stdout.write(`SCORING: (100%)\nPASSED!`);
```

Comment:
maybe we just skip it if no scoring was provided?

Reply:
This is going to happen when 'results.length > 0' and we have scoring, then, the scoring is '100%'
Or I'm not understanding your comment

Reply:
ah yeah, I saw it wrong. Yup, you're right. No action needed here